### DISCUSSION TITLE
Ability to scroll up in daily notes to see future dates

### DISCUSSION THREAD

**Discussion by adhikasp** [athensresearch/athens#887] [Feature Requests]
Feature that I would like to have is ability to scroll up in the "daily notes" view. This will fetch the page for tomorrow (or to the next date that already have content). This scrolling to past or future date should never auto generate page that I never create nor linked before. 

The goal is I can see if I have linked agenda from the past coming up.

Example flow
Case 1:
Today is 2nd Jan. I have daily notes for 1st Jan (link to 3rd Jan) and 2nd Jan.
When I open daily notes and scroll up, it will shows the 3rd Jan page, with bidirectional link to 1st Jan.

Case 2:
Today is 2nd Jan. I have daily notes for 1st Jan (link to 5th Jan) and 2nd Jan.
When I open daily notes and scroll up, it will shows the 5th Jan page, with bidirectional link to 1st Jan.
There should be no auto-generated page for 3rd and 4th Jan because the page is not created yet.

Case 3:
Today is 2nd Jan. I have daily notes for 1st Jan (with no link to future date) and 2nd Jan.
When I open daily notes and scroll up, it should not move upward or provide a text/some feedback that indicate that there is no more content in future date.

See discord discussion https://discord.com/channels/708122962422792194/800439780910563358/823899346012864514

**Comment by lswank** [comment]
Perhaps a stopgap would be to add a dot or other decorator to the calendar/date picker?

I actually worry that scrolling up would create a distraction for people who are newly onboarding to the concept of having daily pages.

**Reply by jefftangx** [reply]
There was a period where you could scroll up to create new pages in the future. People found it really confusing and disorienting, so the UX would have to be worked on :)
